src/Documents.jl

```diff
@@ -266,10 +273,21 @@ function Document(plugins = nothing;
         version = "git:$(Utilities.get_commit_short(root))"
     end
 
+    if working_dir == :build
+        # set working directory to be the same as `build`
+        working_dir = build
```

Comment:
I think this changes behaviour? `pwd` will now always be `build/` right (by default)? Currently, it is `dirname(source_file)`, i.e. potentially different for each page if the pages live in subdirectories. And this is important when at-blocks create images or something that are to be included with the `![]()` syntax, since you need to specify the relative path.

I would maybe store the `working_dir` option as a `Union{Symbol,String}` in `User` and do this check when creating the `Page` objects in `addpage!`.

Reply:
OK, will try that, thanks!

Reply:
Done